README.md

```diff
@@ -235,8 +235,6 @@ new OAuthApp({
 });
 ```
 
-Defaults to `@octokit/oauth-app`'s owne `Octokit` constructor which can be imported separately from `OAuthApp`. It's [`@octokit/core`](https://github.com/octokit/core.js) with the [`@octokit/auth-oauth-user`](https://github.com/octokit/auth-oauth-user.js/#readme) authentication strategy.
-
```

Comment:
this part is actually correct, can revert this change please?

Reply:
Hello @gr2m, I have reverted the change, as requested.